tools/mapping-tester/generate.py

```diff
@@ -25,6 +25,7 @@ def generateCases(setup):
     meshes = setup["general"]["meshes"]
     network = setup["general"].get("network")
     syncmode = setup["general"].get("synchronize", "false")
+    writeMapped = setup["general"].get("writeMapped", "true")
```

Comment:
While the idea or feature is good, I don't really like the name. Maybe rather something like "computeError", "computeAccuracy", "saveSolution" or "saveToFile"? I could imagine that actually an even more cleaner solution would be to directly compute the error in the C++ code. Using the mapping-tester as such, one has the reference solution already provided on the output mesh.

Reply:
`computeAccuracy` sounds good.